+hw/+ptb/Window.m

```diff
@@ -612,14 +612,32 @@ function applyCalibration(obj, cal)
       Screen('LoadNormalizedGammaTable', obj.PtbHandle, gammaTable);
     end
     
-    function c = calibration(obj, dev, lightIn, clockIn, clockOut)
+    function c = calibration(obj, dev, lightIn, clockIn, clockOut, makePlot)
       % Creates a calibration file automatically using the light meter
```

Comment:
can you make explicit this is for gamma calibration somewhere in this docstring?

Reply:
Sure